router/core/factoryresolver.go

```diff
@@ -73,7 +73,6 @@ func NewDefaultFactoryResolver(
 ) *DefaultFactoryResolver {
 
 	defaultHttpClient := &http.Client{
-		Timeout:   transportFactory.DefaultTransportTimeout(),
```

Comment:
`DefaultTransportTimeout` is no longer used anywhere. I suggest to remove it from the interface and delete the implementation.

Reply:
done